src/cli/trn_cli_common.c

```diff
@@ -151,6 +153,27 @@ int trn_cli_parse_net(const cJSON *jsonobj, struct rpc_trn_network_t *net)
 		}
 	}
 	net->switches_ips.switches_ips_len = i;
+	// Cluster gateway is NULL or empty if it is not set in a config map which is a string value
+	net->cluster_gateway = 0;
```

Comment:
use "null" instead of "0"

Reply:
It is an integer which can't be replace with null/NULL.

Reply:
oh i see. I was confused by the comment on line 156.